md-to-html/index.mjs

```diff
@@ -1,20 +1,15 @@
 import { processMarkdown, genResponse } from './core.mjs';
 
 export async function handler(event, context) {
-  if (!event || !event.body) {
-    return genResponse(400, 'Missing event or body');
+  if (!event || !event.markdown) {
+    return genResponse(400, 'Missing event or markdown');
```

Comment:
Missing some context, I am assuming the body is now transformed using API gateway or something similar?